src/runtime/local/kernels/CMakeLists.txt

```diff
@@ -21,10 +21,17 @@ set(CMAKE_POSITION_INDEPENDENT_CODE ON)
 
 # The library of pre-compiled CUDA kernels
 if(USE_CUDA AND CMAKE_CUDA_COMPILER)
+
+    set(CUDA_CODEGEN_CPP_FILES)
+    foreach(i RANGE 0 20)
+        list(APPEND CUDA_CODEGEN_CPP_FILES ${PROJECT_BINARY_DIR}/src/runtime/local/kernels/CUDAkernels_${i}.cpp)
```

Comment:
I don't know why it had to be CUDAkernels_${i}.cpp - the filename you had there before did not work but this did.